cluster-api/Makefile

```diff
@@ -0,0 +1,25 @@
+# Copyright 2018 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+.PHONY: genapi genconversion
+
+all: genapi genconversion
+
+genapi:
+	go get -u github.com/kubernetes-incubator/apiserver-builder/cmd/apiregister-gen
```

Comment:
This means we're always using the tool at head. Is it possible to vendor, build from vendor, and use that copy?